pkg/util/arm/arm_test.go

```diff
@@ -47,28 +47,35 @@ func TestResourceMarshal(t *testing.T) {
   "float": 1.1,
   "array": [
     {
-      "bool": true
+      "bool": true,
+      "tags": null
     }
   ],
   "interface": {
-    "int": 1
+    "int": 1,
+    "tags": null
   },
   "map": {
     "one": {
-      "uint": 1
+      "uint": 1,
+      "tags": null
     },
-    "zero": {}
+    "zero": {
+      "tags": null
+    }
   },
   "ptr": "test",
   "slice": [
     {
-      "float": 1.1
+      "float": 1.1,
+      "tags": null
     }
   ],
   "byte_slice": "dGVzdA==",
   "string": "test",
   "struct": {
-    "string": "test"
+    "string": "test",
+    "tags": null
```

Comment:
Note that `"tags": null` is omited on the top-level (the `Tags` field was overiden by the one from `arm.Resource`), but not on the deeper levels.

Othervise - order of the fields is the same.